griptape/drivers/video_generation/dream_machine_video_generation_driver.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+import logging
+import time
+from typing import TYPE_CHECKING, Any
+
+import requests
+from attrs import Factory, define, field
+
+from griptape.artifacts import VideoArtifact
+from griptape.drivers import BaseVideoGenerationDriver
+from griptape.utils import import_optional_dependency
+
+if TYPE_CHECKING:
+    from lumaai import LumaAI
+
+logger = logging.getLogger(__name__)
+
+
+@define
+class DreamMachineVideoGenerationDriver(BaseVideoGenerationDriver):
+    api_key: str = field(kw_only=True, metadata={"serializable": True})
+    client: LumaAI = field(
+        default=Factory(
+            lambda self: import_optional_dependency("lumaai").LumaAI(auth_token=self.api_key), takes_self=True
+        ),
+        kw_only=True,
+    )
+    params: dict[str, Any] = field(default={}, kw_only=True, metadata={"serializable": True})
```

Comment:
use `factory=dict` here, this will return a single mutable object which is no good